client/me/purchases/manage-purchase/index.jsx

```diff
@@ -173,7 +173,7 @@ const ManagePurchase = React.createClass( {
 			const expiryMoment = this.moment( purchase.expiryMoment );
 			const daysToExpiry = this.moment( expiryMoment.diff( this.moment() ) ).format( 'D' );
 
-			return this.translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s days. ',
+			return this.translate( '%(purchaseName)s will expire and be removed from your site in %(expiry)s days. ',
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( '%(purchaseName)s will expire and be removed from your site %(expiry)s.' )` *ES Score: 5.16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).